### PR TITLE
[WEEKLY RELEASE] HotShot - rc-0.5.71

### PR DESCRIPTION
**Pull request by rob-maron**

### CODE REVIEW

File: marketplace-builder/src/builder.rs [hunk 1]
```diff
@@ -181,6 +181,7 @@ impl BuilderConfig {
         let proxy_global_state = ProxyGlobalState::new(
             global_state.clone(),
             (builder_key_pair.fee_account(), builder_key_pair.clone()),
+            Duration::from_secs(5),
```
Comment (rob-maron) [inline marketplace-builder/src/builder.rs:184]: @QuentinI What should I set this to?

Comment (QuentinI) [reply marketplace-builder/src/builder.rs:184]: That's where `max_api_timeout_duration` should go

File: builder/src/non_permissioned.rs [hunk 2]
```diff
@@ -11,7 +11,7 @@ use async_compatibility_layer::{
 use async_std::sync::{Arc, RwLock};
 use espresso_types::{
     eth_signature_key::EthKeyPair, v0_3::ChainConfig, FeeAmount, L1Client, NodeState, Payload,
-    SeqTypes, SequencerVersions, ValidatedState,
+    SeqTypes, SeqVersions, ValidatedState,
```
Comment (tbro) [inline builder/src/non_permissioned.rs:14]: Can we keep `SequencerVersions`?

**Review by tbro — CHANGES_REQUESTED** [review]

File: builder/src/non_permissioned.rs (hunk 2 above)
Comment (rob-maron) [reply builder/src/non_permissioned.rs:14]: I have no preference, I think it is a consequence of that big merge. Let me switch it

File: builder/src/non_permissioned.rs [hunk 3]
```diff
@@ -193,20 +193,17 @@ impl BuilderConfig {
...
-                )
-                .await;
+            let res = run_non_permissioned_standalone_builder_service::<_, SequencerVersions>(
+                da_sender,
+                qc_sender,
+                decide_sender,
+                events_url,
+                global_state_clone,
+            )
+            .await;
```
Comment (rob-maron) [inline builder/src/non_permissioned.rs:206]: Here

File: builder/src/permissioned.rs [hunk 4]
```diff
@@ -503,14 +501,15 @@ impl<N: ConnectedNetwork<PubKey>, P: SequencerPersistence, Ver: StaticVersionTyp
...
         // spawn the builder service
         async_spawn(async move {
             run_permissioned_standalone_builder_service(
-                tx_sender,
                 da_sender,
                 qc_sender,
                 decide_sender,
                 hotshot_handle_clone,
+                global_state_clone,
             )
```
Comment (rob-maron) [inline builder/src/permissioned.rs:513]: This may also be relevant

**Review by tbro — APPROVED** [review]

**Pull request merged**